libraries/AP_Notify/ExternalLED.h

```diff
@@ -23,6 +23,9 @@
 
 #include "NotifyDevice.h"
 
+#define HIGH 1
+#define LOW 0
```

Comment:
Into ExternalLED.cpp HAL_GPIO_LED_OFF, defined into AP_HAL/AP_HAL_Boards.h, don't see the definition of HIGH and LOW. Is the same thing of AP_BoardLED.cpp (HIGH and LOW are defined into AP_BoardLED.h)